src/python/dxpy/app_builder.py

```diff
@@ -904,15 +904,15 @@ def _create_app(applet_or_regional_options, app_name, src_dir, publish=False, se
         if authorized_users_to_add:
             dxpy.api.app_add_authorized_users(app_id, input_params={'authorizedUsers': list(authorized_users_to_add)})
         if skip_adding_public:
-            logger.warn('the app was NOT made public as requested in the dxapp.json. To make it so, run "dx add users app-%s PUBLIC".' % (app_spec["name"],))
+            logger.warn('the app was NOT made public as requested in the app spec. To make it so, run "dx add users app-%s PUBLIC".' % (app_spec["name"],))
```

Comment:
@mhrvol @r-i-v-a the messages in app_builder are updated. Thanks for the suggestion!

Reply:
I thought @mhrvol 's suggestion was to go the other way, and use dxapp.json / dxworkflow.json throughout? dxapp.json is used often enough in our user-facing documentation that it should be clear to the user what it refers to https://documentation.dnanexus.com/developer/apps/app-metadata , maybe clearer than calling it "app spec."

Reply:
Ok, I defer to @commandlinegirl 's comment on the other thread, so it can stay as "app / workflow spec."